### DISCUSSION TITLE
阿里云跟文件夹下再创建文件夹，文件夹内容之间移动会报错，但是移动成功了

### DISCUSSION THREAD

**Discussion by mlsqq** [AlistGo/alist#835] [Q&A]
![photo_2022-03-28_12-50-31](https://user-images.githubusercontent.com/85686342/160328560-d3b392de-4430-46f3-b091-1dfa524f086c.jpg)
如图所示

**Comment by xhofe** [comment]
正常的api变动。

**Reply by mlsqq** [reply]
谢谢回答！我提个建议，移动文件点确定后弹出其他颜色通知，并返回，目前现在调整之后得手动关闭调整文件夹

**Reply by xhofe** [reply]
正常就是这样的，只是阿里的api返回的数据改了，所以判定失败。下个版本会跟随它的api变动
